### PR TITLE
Vertex SDK - LLM - Added tuning samples for the `ChatModel` (`chat-bison`) and `CodeChatModel` (`codechat-bison`)

### PR DESCRIPTION
**Pull request by Ark-kun**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: generative_ai/tune_chat_model.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      location: GCP Region, used to initialize vertexai
+      training_data: GCS URI of jsonl file or pandas dataframe of training data
+      train_steps: Number of training steps to use when tuning the model.
+    """
+    vertexai.init(project=project_id, location=location, credentials=credentials)
+    model = CodeChatModel.from_pretrained("codechat-bison@001")
+
+    tuning_job = model.tune_model(
+        training_data=training_data,
+        # Optional:
```
Comment (dandhlee) [inline generative_ai/tune_chat_model.py:58]: Remove unused comment.

**Review by dandhlee — CHANGES_REQUESTED** [review]
Reviewing for Python & sample style.

**Review by Ttris31 — APPROVED** [review]

**Comment by Ark-kun** [comment]
Thank you for the review. I'll try to implement the feedback.

I've tried to keep the samples consistent. These new samples are just copies of the existing samples created by the DPE team with model names changed. Perhaps the DPE team can initiate a repo-wide refactoring based on this feedback.

File: generative_ai/tune_chat_model.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        train_steps=train_steps,
+        tuning_job_location="europe-west4",
+        tuned_model_location=location,
+    )
+    tuned_model = tuning_job.get_tuned_model()
+    # [END aiplatform_sdk_tune_code_chat_model]
```
Comment (msampathkumar) [inline generative_ai/tune_chat_model.py:64]:
```
    # [END aiplatform_sdk_tune_code_chat_model]
```

This does not match the start tag. Please check.

**Pull request closed without merging**